projects/packages/videopress/src/client/block-editor/blocks/video/components/player/types.ts

```diff
@@ -10,16 +10,6 @@ export type PlayerProps = {
 	attributes: VideoBlockAttributes;
 	setAttributes: ( attributes: VideoBlockAttributes ) => void;
 	scripts: string[];
-	preview: {
-		height: number;
-		html: string;
-		provider_name: 'VideoPress';
```

Comment:
Hi, there. Those props are defined in the`/oembed/1.0/proxy` endpoint response body:

<img width="877" alt="image" src="https://user-images.githubusercontent.com/77539/224841159-fd773c95-ba14-4daa-b7f9-bfb9e3422750.png">

Are you getting TS issues? Is that why you want to make them optional?

Reply:
Yes and no. I was getting TS issues but I wasn't sure why those props were there in ‘player/types.ts’ since they are not use in the ’Player’ component. 

But since the main idea here is to have the "source of truth" type higher up from the player component directory then those should be there. Thanks for pointing out the oembed response!

Do you see an issue making those optional props since they are not required in the player?

Reply:
I'm unsure if you are facing other issues, but AFAICS, removing the `scripts` prop from the player component and thus from the types won't generate any case, at least in this context.

The props should now be optional since that's what we expect from the endpoint response. Being said that, it's up to you, actually :-)